src/parser/tptp/Tptp.g

```diff
@@ -769,7 +769,8 @@ tffLetTermDefn[CVC4::Expr& lhs, CVC4::Expr& rhs]
 
 tffLetTermBinding[std::vector<CVC4::Expr>& bvlist, CVC4::Expr& lhs, CVC4::Expr& rhs]
   : term[lhs] EQUAL_TOK term[rhs]
-    { PARSER_STATE->checkLetBinding(bvlist, lhs, rhs, false);
+    { std::sort(bvlist.begin(), bvlist.end());
```

Comment:
Andy, is this needed?

Reply:
I'm 99% positive the reordering is not needed, you can remove.